### PR TITLE
OCPBUGS-6964: SME feedback on yaml file

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to main, 4.13 and 4.12
- [Jira](https://issues.redhat.com/browse/OCPBUGS-6964)
- [Preview](https://55912--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#installation-bare-metal-agent-installer-config-yaml_preparing-to-install-with-agent-based-installer)
- @zaneb ptal

### CODE REVIEW

File: modules/installation-bare-metal-agent-installer-config-yaml.adoc [hunk 1]
```diff
@@ -14,68 +14,53 @@ You can customize the `install-config.yaml` file to specify more details about y
...
-<11> You must set the platform to `none` for a single-node cluster. You can also set the platform to `vSphere` and `baremetal`.
+<7> The subnet prefix length to assign to each individual node. For example, if `hostPrefix` is set to `23`, then each node is assigned a `/23` subnet out of the given `cidr`, which allows for 510 (2^(32 - 23) - 2) pod IP addresses. If you are required to provide access to nodes from an external network, configure load balancers and routers to manage the traffic.
+<8> The cluster network plugin to install. The supported values are `OVNKubernetes` (default value) and `OpenShiftSDN`.
+<9> The IP address pool to use for service IP addresses. You can enter only one IP address pool. This block must not overlap with existing physical networks. If you need to access the services from an external network, configure load balancers and routers to manage the traffic.
+<10> You must set the platform to `none` for a single-node cluster. You can set the platform to `vSphere` and `baremetal` for multi-node clusters.
```
Comment (zaneb) [inline modules/installation-bare-metal-agent-installer-config-yaml.adoc:59]: `vSphere` should be `vsphere`. Note that `none` is not currently allowed for multinode clusters, so the only choice in that case is between `baremetal` and `vsphere`.

```suggestion
<10> You must set the platform to `none` for a single-node cluster. You can set the platform to either `vsphere` or `baremetal` for multi-node clusters.
```

Comment (Amrita42) [reply modules/installation-bare-metal-agent-installer-config-yaml.adoc:59]: ok noted

Comment (Amrita42) [reply modules/installation-bare-metal-agent-installer-config-yaml.adoc]: Implemented. ptal

File: modules/installation-bare-metal-agent-installer-config-yaml.adoc [hunk 2]
```diff
@@ -14,68 +14,53 @@ You can customize the `install-config.yaml` file to specify more details about y
...
-<11> You must set the platform to `none` for a single-node cluster. You can also set the platform to `vSphere` and `baremetal`.
+<7> The subnet prefix length to assign to each individual node. For example, if `hostPrefix` is set to `23`, then each node is assigned a `/23` subnet out of the given `cidr`, which allows for 510 (2^(32 - 23) - 2) pod IP addresses. If you are required to provide access to nodes from an external network, configure load balancers and routers to manage the traffic.
+<8> The cluster network plugin to install. The supported values are `OVNKubernetes` (default value) and `OpenShiftSDN`.
+<9> The IP address pool to use for service IP addresses. You can enter only one IP address pool. This block must not overlap with existing physical networks. If you need to access the services from an external network, configure load balancers and routers to manage the traffic.
+<10> You must set the platform to `none` for a single-node cluster. You can set the platform to either `vsphere` and `baremetal` for multi-node clusters.
```
Comment (zaneb) [inline modules/installation-bare-metal-agent-installer-config-yaml.adoc:59]: s/and/or/
otherwise lgtm

Comment (Amrita42) [reply modules/installation-bare-metal-agent-installer-config-yaml.adoc:59]: oops I missed that , _or_ goes with _either_ . pushing this change.

**Review by snarayan-redhat — COMMENTED** [review]
nice work. LGTM!

**Pull request merged**